network/indy-pool.dockerfile

```
@@ -105,4 +105,4 @@ COPY network/indy-cli-config.json /etc/indy/indy-cli-config.json
 
 EXPOSE 9701 9702 9703 9704 9705 9706 9707 9708
 
-CMD ["/usr/bin/supervisord"]
\ No newline at end of file
+CMD ["/usr/bin/supervisord"]
```

This seems unrelated?

```suggestion
CMD ["/usr/bin/supervisord"] 
```